### PR TITLE
Adds more tests

### LINKED ISSUE #227: Adds more tests
- Tests for sales storage
- Tests for unarchive command
- Tests for FindByTag command
- Tests for FindByTagToday command
- Tests for ArchiveCommandParser
- Tests for UnarchiveCommandParser
- Tests for FindTagCommandParser

### PR DESCRIPTION
**Pull request by yanbingtao**
- Tests for sales storage
- Tests for unarchive command
- Tests for FindByTag command
- Tests for FindByTagToday command
- Tests for ArchiveCommandParser
- Tests for UnarchiveCommandParser
- Tests for FindTagCommandParser

### CODE REVIEW

File: src/main/java/seedu/address/model/SalesBook.java [hunk 1]
```diff
@@ -113,7 +114,10 @@ public void sortRecord() {
         record.sort();
     }
 
-    //// util methods
```
Comment (ureshiiYing) [inline src/main/java/seedu/address/model/SalesBook.java:116]: Could you restore this line? to indicate the start of the util methods.

File: src/test/java/seedu/address/testutil/TypicalSalesRecordEntries.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
+package seedu.address.testutil;
```
Comment (ureshiiYing) [inline src/test/java/seedu/address/testutil/TypicalSalesRecordEntries.java:1]: hmm, I have also added this file, but differently... Will it affect your tests greatly, if mine could be used instead?

File: src/main/java/seedu/address/model/SalesBook.java (hunk 1 above)
Comment (yanbingtao) [reply src/main/java/seedu/address/model/SalesBook.java:116]: Sure

File: src/test/java/seedu/address/testutil/TypicalSalesRecordEntries.java (hunk 2 above)
Comment (yanbingtao) [reply src/test/java/seedu/address/testutil/TypicalSalesRecordEntries.java:1]: okay can, then I will adjust my code to use your TypicalSalesRecordEntries.java file

**Review by Diwu-Yi — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/model/SalesBook.java (hunk 1 above)
Comment (ureshiiYing) [reply src/main/java/seedu/address/model/SalesBook.java:116]: It should be restored below the setSalesRecordEntry method.

**Review by WM71811 — APPROVED** [review]
LGTM

**Pull request merged**